package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bigcommerce/checkout",
-  "version": "1.112.0",
+  "version": "1.110.1",
```

Comment:
@andriiVitvitskyi1990 checkout version is fully managed by circle, no need  to modify